app/services/injection_response_service.rb

```diff
@@ -18,7 +18,7 @@ def run!
   private
 
   def slack
-    @slack ||= SlackNotifier.new(@channel)
+    @slack ||= SlackNotifier.new(@channel, formatter: SlackNotifier::Formatter::Injection.new)
```

Comment:
nit - `notifier` or `slack_notifier` probably a clearer name since `slack` is the class name